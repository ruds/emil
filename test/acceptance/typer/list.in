[];
[1, 2, 3, 4];
[(1, "foo"), (2, "bar"), (3, "baz")];
[1, "foo"];
[] : int list;
[1, 2, 3, 4] : bigint list;
[1, 2, 3, 4] : string list;
