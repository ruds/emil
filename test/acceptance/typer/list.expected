@0001:
val it : 'a list
@0002:
val it : bigint list
@0003:
val it : (bigint * string) list
@0004: ERROR
@0005:
val it : int list
@0006:
val it : bigint list
@0007: ERROR
@0008:
